### PR TITLE
Add method to get output directories

### PR DESCRIPTION
**Pull request by josselinchevalay**
that can permit to clean intermediate and output
directories.

### CODE REVIEW

File: src/main/groovy/com/ullink/Msbuild.groovy [hunk 1]
```diff
@@ -240,4 +240,18 @@ class Msbuild extends ConventionTask {
         }
         return cmdParameters
     }
+
+    def getOutputDirs() {
+        def HashMap<String, Object> cmdParameters = getInitProperties()
+        def HashMap<String, String> dirs = ['OutputPath': '.\\build\\bin\\' , 'IntermediateOutputPath' : '.\\build\\obj\\']
```
Comment (gluck) [inline src/main/groovy/com/ullink/Msbuild.groovy]: These output folders can have any value in the csproj files, these default/hardcoded paths are unlikely to work except with the most basic projects, don't think we can merge this.

Comment (ngyukman) [reply src/main/groovy/com/ullink/Msbuild.groovy]: we should able to get the property from project properties or task parameters if specified

Comment (josselinchevalay) [reply src/main/groovy/com/ullink/Msbuild.groovy]: do you have an example @ngyukman maybe we can get that also in csproj file no ?

File: src/main/groovy/com/ullink/Msbuild.groovy [hunk 2]
```diff
@@ -240,4 +245,17 @@ class Msbuild extends ConventionTask {
         }
         return cmdParameters
     }
+
+    @OutputDirectories
+    def getOutputDirs() {
+        def paths = []
+        def cmdParameters = getInitProperties()
```
Comment (gluck) [inline src/main/groovy/com/ullink/Msbuild.groovy]: AFAIK init properties are optional prop values specified in the gradle build, they may not be provided.
Doubt this works.

**Pull request closed without merging**